plugins/modules/nxos_pim_interface.py

```diff
@@ -527,6 +536,7 @@ def main():
         dr_prio=dict(type="str"),
         hello_auth_key=dict(type="str", no_log=True),
         hello_interval=dict(type="int"),
+        hello_interval_ms=dict(type="bool"),
```

Comment:
Should you default this to `False`?

Reply:
+1

Reply:
Oh, actually setting default: False to hello_interval_ms would mean that the the "required_by" condition that ensures if hello_interval_ms is set explicitly then hello_interval is also specified would fail.